utils/_context/_scenarios/auto_injection.py

```diff
@@ -73,11 +68,6 @@ def __init__(
         if include_centos_7_amd64:
             self.required_vms.append(Centos7amd64())
 
-    def session_start(self):
```

Comment:
If we move this code to get_warmups, the method is going to execute several times if we use xdist.

Reply:
Check this: https://github.com/DataDog/system-tests/pull/2970/files

Reply:
Good catch, I've added a `is_main_worker` field. Can you tell me if I've correctly understood what to run here ? 

https://github.com/DataDog/system-tests/pull/2969/files#diff-6bf522167f4c94fdbfb99ce57f86722e8b180e2364bb4315f7f3beb1be479be0R134-R141

Reply:
Added some comments